src/shared/static/shared/font/fontello/config.json

```diff
@@ -631,6 +631,8 @@
       "search": [
         "thought"
       ]
+<<<<<<< HEAD
+=======
```

Comment:
Looks like merge had problems. I think if you copy the fontello directory from current development it should work.

Reply:
Sorry, thought I got all of these - too early in the morning to do merges I guess. Fixing now.